DataLayer/BatteryFaultsData/I_BatteryFaultsData.h

```diff
@@ -38,8 +38,8 @@ class I_BatteryFaultsData : public QObject
     virtual unsigned short getErrorFlag() const = 0;
 
     /* BatteryFaults Sets */
-    void setPackageID(const unsigned char& packageID) = 0;
-    void setErrorFlag(const unsigned char& errorFlag) = 0;
+    virtual void setPackageID(const unsigned char& packageID) = 0;
```

Comment:
I think a SerialDefines file would be best, it's more inline how the project has been coded so far.